docs/guidelines.md

```diff
@@ -210,15 +210,15 @@ For example:
  *  * theMetadata: Description.
  *
  * @return {angular.Directive} The directive specs.
- * @htmlAttribute {ol.Map} ngeo-control-map The map.
+ * @htmlAttribute {ol.Map} ngeo-misc-control-map The map.
  * @ngInject
  * @ngdoc directive
  * @ngname ngeoControl
  */
-ngeo.controlDirective = function() {
+ngeo.misc.controlComponent.component_ = function() {
   // …
 };
-ngeo.module.directive('ngeoControl', ngeo.controlDirective);
+ngeo.misc.controlComponent.directive('ngeoMiscControl', ngeo.misc.controlComponent.component_);
```

Comment:
I think you should keep 'ngeoControl'.

Reply:
Yes, thanks, I don't know why I've do that

Reply:
Otherwise LGTM.

Reply:
Ahhh... but it's the guidlines. Does it matter ?
Perhaps it's even better to remove them...

Reply:
The guideline describes how to deal with the Closure compiler symbols but says nothing about renaming the names used in Angular.

Reply:
Ah, I understand what you mean now! ^^
OK, we can merge as it is.